src/sentry/api/serializers/models/integration.py

```diff
@@ -58,11 +59,18 @@ def serialize(self, obj, attrs, user, include_config=True):
         else:
             data.update({"configOrganization": install.get_organization_config()})
 
+            # Query param "action" only attached in TicketRuleForm modal.
+            if self.params.get("action") == "create":
+                data["createIssueConfig"] = install.get_create_issue_config_no_params()
```

Comment:
@mgaeta Is this some sort of optimization so we don't have to fetch the issue config later? Trying to understand how this is different than the create issue from on the issue details page.